src/Command.js

```diff
@@ -182,6 +183,13 @@ export default class Command {
       if (this.getOptions().includeFilteredDependencies) {
         this.filteredPackages = PackageUtilities.addDependencies(this.filteredPackages, this.packageGraph);
       }
+      if (this.flags.onlyUpdated) {
```

Comment:
Well, consider that those other commands really have no use for this flag, and might even be considered broken if they were to adhere to it. I would recommend a separate shared file to encapsulate, yes.